Kitodo/src/main/java/org/kitodo/export/ExportMets.java

```diff
@@ -63,12 +63,15 @@
     public void startExport(Process process) throws IOException, DAOException {
         User user = ServiceManager.getUserService().getAuthenticatedUser();
         URI userHome = ServiceManager.getUserService().getHomeDirectory(user);
+        boolean exported = process.isExported();
+        process.setExported(true);
         boolean exportSucessfull = startExport(process, userHome);
         if (exportSucessfull) {
-            process.setExported(true);
-        }
-        if (Objects.nonNull(process.getParent())) {
-            startExport(process.getParent());
+            if (Objects.nonNull(process.getParent())) {
+                startExport(process.getParent());
+            }
+        } else {
+            process.setExported(exported);
```

Comment:
A save to DB is missing here (and above in ExportDms).
Please adjust ExportMetsIT to test reload from DB.